webbpsf/opds.py

```diff
@@ -1336,7 +1359,7 @@ def _apply_hexikes_to_seg(self, segment, hexike_coeffs, debug=False):
         apmask = np.ones_like(Xc)  # by construction, we're only evaluating this for the good pixels
 
         hexikes = zernike.hexike_basis_wss(x=Xc, y=Yc, nterms=len(hexike_coeffs),
-                                           aperture=apmask)
+                                           aperture=apmask, npix=self.npix) #FIXME: KJB do we need to pass npix here?
```

Comment:
I think not  needed to pass `npix` in this case, because Xc, Yc, and apmask all encode the number of pixels sufficiently. The `hexike_basis_wss` function will use the X and Y coordinates supplied so that sets the number of pixels in the output.